features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/model/RoomListRoomSummary.kt

```diff
@@ -35,3 +36,13 @@ data class RoomListRoomSummary(
     val hasRoomCall: Boolean,
     val isDm: Boolean,
 )
+
+fun RoomListRoomSummary.isTimestampHighlighted(): Boolean {
```

Comment:
Makes it a computed val instead

Reply:
958e97f89c58a251954429e776c48ad0c649809e